arches_lingo/src/arches_lingo/components/scheme/report/SchemeNamespace.vue

```diff
@@ -19,14 +28,15 @@ import type {
 
 const toast = useToast();
 const { $gettext } = useGettext();
-const schemeInstance = ref<SchemeInstance>();
+const schemeInstance = ref<SchemeInstance>({});
 const route = useRoute();
+const router = useRouter();
 
 defineProps<{
     mode?: DataComponentMode;
 }>();
 
-const emit = defineEmits([OPEN_EDITOR, "updated"]);
+const emit = defineEmits([OPEN_EDITOR, UPDATED]);
```

Comment:
IMO should be `emits` to match the rest of the app ( OR the rest of the app should be `emit` to bring inline with this and the Vue docs  😄 )